src/transaction/log_page_buffer.c

```diff
@@ -4499,10 +4499,17 @@ logpb_flush_pages (THREAD_ENTRY * thread_p, const LOG_LSA * flush_lsa)
       // *INDENT-OFF*
       if (get_server_type () == SERVER_TYPE_TRANSACTION && ts_Gl->is_page_server_connected ())
 	{
-	  log_Gl.wait_flushed_lsa (*flush_lsa);
 	  if (prm_get_bool_value (PRM_ID_ER_LOG_COMMIT_CONFIRM))
 	    {
-	      _er_log_debug (ARG_FILE_LINE, "Page server committed LSA = %lld|%d.\n", LSA_AS_ARGS (&log_Gl.m_max_ps_flushed_lsa));
+	      _er_log_debug (ARG_FILE_LINE, "Wait until consenesus LSA gets larger than or equal to %lld|%d.\n", LSA_AS_ARGS (flush_lsa));
+	    }
+	  
+          log_Gl.wait_flushed_lsa (*flush_lsa);
+	  
+          if (prm_get_bool_value (PRM_ID_ER_LOG_COMMIT_CONFIRM))
+	    {
+              std::lock_guard<std::mutex> lock_guard (log_Gl.m_ps_lsa_mutex);
+	      _er_log_debug (ARG_FILE_LINE, "Page server committed LSA = %lld|%d.\n", LSA_AS_ARGS (&log_Gl.m_ps_consensus_flushed_lsa));
```

Comment:
```suggestion
	      _er_log_debug (ARG_FILE_LINE, "Page server consensus LSA = %lld|%d.\n", LSA_AS_ARGS (&log_Gl.m_ps_consensus_flushed_lsa));
```
maybe it is better to change "committed" to "consensus" or something.

Reply:
Hmm.. my concern was some regression tests would already use the msg like "COMMIT CONFIRM". However, I can't find "server committed" string in tests using `grep`. So, I guess I can revise it.

Reply:
We can adapt the regression test. I think it is more important - for the long term - to have meaningful /debugerror messages. :)
So, if it's more proper to reword the message, we can do it.

Reply:
OK, I'll revise debug loggings. Let's adapt test cases if needed.